### ISSUE TITLE
[BUG] Not loading tokens

### ISSUE THREAD

**Issue by realpaaao** [vultisig/vultisig-ios#641]
**Describe the bug**
On version 28 I am not able to view and add tokens. Possible to reproduce on Mac, iPad, iPhone

**To Reproduce**
1. Go to any chain with tokens
2. select tokens 
3. See error

**Expected behavior**
See normal token lsit for repective chain

**Screenshots**
![image](https://github.com/vultisig/vultisig-ios/assets/167348323/9a9101c9-36f8-4ed4-9c90-a6a619953d09)

Possi

**Comment by enriquesouza** [comment]
Yes, this is being fixed already, because these tokens only appear when searching. 
I've put a message. Start typing ...
<img width="450" alt="image" src="https://github.com/vultisig/vultisig-ios/assets/1700699/27161b19-2bc7-4cdf-b8ed-46ac2ced5c53">

**Comment by johnnyluo** [comment]
This also get fixed in https://github.com/vultisig/vultisig-ios/pull/672
